src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -10,8 +10,11 @@ import org.spongycastle.math.ec.{ECAlgorithms, ECPoint, ECCurve}
 import org.spongycastle.util.BigIntegers._
 
 object ECDSASignature {
+
   val RAndSLength = 32
```

Comment:
I think that 2 separate values will be better, I do not know how to correctly name value that is length of both